projects/igniteui-angular/src/lib/time-picker/time-picker.pipes.ts

```diff
@@ -1,5 +1,6 @@
 import { Pipe, PipeTransform, Inject } from '@angular/core';
 import { IGX_TIME_PICKER_COMPONENT, IgxTimePickerBase } from './time-picker.common';
+import { IgxTimePickerComponent } from './time-picker.component';
 
 
```

Comment:
This entire file can be deleted, The date time editor will take care of updating the input etc.